scipy/integrate/_ivp/bdf.py

```diff
@@ -89,9 +89,7 @@ class BDF(OdeSolver):
         shape (n,). Alternatively it can have shape (n, k); then ``fun``
         must return an array_like with shape (n, k), i.e. each column
         corresponds to a single column in ``y``. The choice between the two
```

Comment:
Should not this part be updated as well?

Reply:
Oops!